MapboxNavigation/RouteMapViewController.swift

```diff
@@ -505,7 +507,7 @@ func defaultFeedbackHandlers(source: FeedbackSource = .user) -> (send: FeedbackV
             self.navigationView.floatingStackView.alpha = 1.0
         }
         
-        let complete: (Bool) -> Void = { self.navigationView.endOfRouteView?.isHidden = true; completion?($0)}
+        let complete: (Bool) -> Void = { self.navigationView.endOfRouteView?.isHidden = true; self.unembedEndOfRoute(); completion?($0)}
```

Comment:
this was a bad habit for me to get into in the first place when it was two commands. Now that it's three, we should really break this out into a multiline closure.